app/modules/individuals/models.py

```diff
@@ -151,6 +163,19 @@ def get_elasticsearch_schema(cls):
 
         return ElasticsearchIndividualSchema
 
+    @classmethod
```

Comment:
Would help to have a comment about why this was needed. Right now I know but in 2 months time ... I'll have forgotten, frankly I may have forgotten by next week. My brain seems very teflon for ES stuff

Reply:
does the comment go here and/or everywhere else it is used?  (currently only also [previously] in encounters/models.py)

i put it right under `def get_elasticsearch_schema()` because that is very similar -- an elasticsearch override that does special stuff for elasticsearch.   so yeah i can put a comment in this models.py but be warned its likely to sneak in other places!  ES is sneaky that way!